akka-http-core/src/test/scala/akka/http/impl/engine/client/TlsEndpointVerificationSpec.scala

```diff
@@ -47,6 +51,12 @@ class TlsEndpointVerificationSpec extends AkkaSpec("""
         e shouldBe an[Exception]
```

Comment:
> In this file should accept certificates signed by known CA fails now

Which side fails, the client-side or the server-side? HTTPS certificates are usually only checked on the client side so if there's no check to disable it on the server side this may be the reason it fails.